doc-tool/src/dotty/tools/dottydoc/staticsite/Site.scala

```diff
@@ -256,7 +256,7 @@ case class Site(
       blogposts.foreach { file =>
         val BlogPost.extract(year, month, day, name, ext) = file.getName
         val sourceFile = toSourceFile(file)
-        val date = s"$year-$month-$day 00:00:00"
+        val date = s"$month-$day-$year"
```

Comment:
Oh, that really isn't much. Its because we have `date: "%b-%d-%Y" ` in the layouts. Do I change it?